db/warehouse/migrate/20250513132551_add_fy2026_columns.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+class AddFy2026Columns < ActiveRecord::Migration[7.1]
+  def change
+    add_column :Client, :Sex, :integer
+    add_column :Enrollment, :MentalHealthConsultation, :integer
```

Comment:
I wonder if we should use small ints? I think the savings would be very small (2 bytes per row).
```suggestion
    add_column :Client, :Sex, :integer, limit: 2
    add_column :Enrollment, :MentalHealthConsultation, :integer, limit: 2
```

Reply:
In this case I doubt it is worth it, those aren't high cardinality tables.  I believe we have done some tuning on the larger tables.